src/_gettsim/taxes/lohnst.py

```diff
@@ -132,7 +132,7 @@ def vorsorge_krankenv_option_b_ab_2019(
         * (
             sozialv_beitr_params["beitr_satz"]["ges_krankenv"]["ermäßigt"] / 2
             + ges_krankenv_zusatzbeitr_satz / 2 / 100
```

Comment:
Just stumbled across this, nothing to do with this PR... Why the ` / 100`? Do you remember, @JakobWegmann ?

Reply:
A matter of taste. Generally, I tried to be consistent with the "Programmablaufplan" from BMF and they have the "Krankenversicherungszusatzbeitrag" in percent.

![grafik](https://github.com/iza-institute-of-labor-economics/gettsim/assets/37152510/eb20470f-559a-4e37-98c1-e9255e7a01ad)

Should I change that when I add the 2025 test cases for the withholding tax?

Reply:
Thanks for the clarification! I think we (@marvinimmesberger, that is :wink:) can just change it as part of this PR. Since all other social insurance contribution rates are specified as rates, I think consistency within GETTSIM is the more important objective!

Reply:
In the parameter files we already use rates (e.g. the Zusatzbeitrag `mean_zusatzbeitrag` in `src/_gettsim/parameters/sozialv_beitr.yaml` for 2018 is 0.01). The function that creates the Zusatzbeitrag (which is later divided by 100 in the function cited above) is `ges_krankenv_zusatzbeitr_satz_from_mean_zusatzbeitrag`. This function is used on other occasions without division by 100 (e.g. in `ges_krankenv_beitr_satz_arbeitnehmer`). 

Are you sure that this is not a mistake in the Lohnsteuer module? @JakobWegmann

Reply:
Ah just saw, that the Zusatzbeitrag is part of the test inputs in percent! I'll change those and remove the division by 100.

Reply:
Thank you!